tests/API_test.go

```diff
@@ -0,0 +1,165 @@
+package tests
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/himanshuo/gameoflife/models"
+	"net/http"
+	"strings"
+	"testing"
+)
+
```

Comment:
func checkError(t testing, msg String, args ...interface{}){

}
http://stackoverflow.com/a/7053871/4710047 for function name

func (c *common) Errorf(format string, args ...interface{}) {
   366		c.log(fmt.Sprintf(format, args...))
   367		c.Fail()
   368	}